package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.2.0",
   "description": "A tiny math expression library with only the stuff I personally care about. Deal with it.",
   "main": "lib/tinymath.js",
+  "module": "lib/tinymath.mjs",
```

Comment:
I realize `jsnext:main` might be deprecated or whatever, but it's still the only field that eslint-import-plugin supports. can we get both fields added here?

Reply:
Sure